*** /Library/WebServer/svgweb/svgweb/samples/svg-uncompressed.js	2009-11-05 00:04:17.000000000 +0100
--- Desktop/svgweb/src/svg-uncompressed.js	2009-10-12 19:51:52.000000000 +0200
***************
*** 1839,1852 ****
    
    /** Intercepts setting up window.onload events so we can delay firing
        them until we are done with our internal work. */
!    _interceptOnloadListeners: function() {
      if (window.addEventListener) {
        window._addEventListener = window.addEventListener;
        window.addEventListener = function(type, f, useCapture) {
!         if (type == 'svgload') {
!             svgweb.addOnLoad(f);
!         } else {
            return window._addEventListener(type, f, useCapture);
          }
        }
      }
--- 1839,1852 ----
    
    /** Intercepts setting up window.onload events so we can delay firing
        them until we are done with our internal work. */
!   _interceptOnloadListeners: function() {
      if (window.addEventListener) {
        window._addEventListener = window.addEventListener;
        window.addEventListener = function(type, f, useCapture) {
!         if (type != 'load') {
            return window._addEventListener(type, f, useCapture);
+         } else {
+           svgweb.addOnLoad(f);
          }
        }
      }
***************
*** 1854,1863 ****
      if (isIE && window.attachEvent) {
        window._attachEvent = window.attachEvent;
        window.attachEvent = function(type, f) {
!         if (type == 'onsvgload') {
!             svgweb.addOnLoad(f);
!         } else {
            return window._attachEvent(type, f);
          }
        }
      }
--- 1854,1863 ----
      if (isIE && window.attachEvent) {
        window._attachEvent = window.attachEvent;
        window.attachEvent = function(type, f) {
!         if (type != 'onload') {
            return window._attachEvent(type, f);
+         } else {
+           svgweb.addOnLoad(f);
          }
        }
      }
***************
*** 2410,2416 ****
          if (type == 'load') {
            this._listeners.push(listener);
          } else if (!addEventListener) { // IE
!           this.attachEvent('onsvg' + type, listener);
          } else { // W3C
            _addEventListener(type, listener, useCapture);
          }  
--- 2410,2416 ----
          if (type == 'load') {
            this._listeners.push(listener);
          } else if (!addEventListener) { // IE
!           this.attachEvent('on' + type, listener);
          } else { // W3C
            _addEventListener(type, listener, useCapture);
          }  
***************
*** 3138,3144 ****
        // Issue 219: "body.onload not fired for SVG OBJECT"
        // http://code.google.com/p/svgweb/issues/detail?id=219
        var self = this;
!       this._objNode.addEventListener('svgload', function() {
          var win = self._objNode.contentDocument.defaultView;
          self._onObjectLoad(self._objNode._svgFunc, win);
        }, false);
--- 3138,3144 ----
        // Issue 219: "body.onload not fired for SVG OBJECT"
        // http://code.google.com/p/svgweb/issues/detail?id=219
        var self = this;
!       this._objNode.addEventListener('load', function() {
          var win = self._objNode.contentDocument.defaultView;
          self._onObjectLoad(self._objNode._svgFunc, win);
        }, false);
***************
*** 5291,5297 ****
      'onfocusin', 'onfocusout', 'onactivate', 'onclick', 'onmousedown',
      'onmouseup', 'onmouseover', 'onmousemove', 'onmouseout', 'onload',
      'onunload', 'onabort', 'onerror', 'onresize', 'onscroll', 'onzoom',
!     'onbegin', 'onend', 'onrepeat', 'onsvgload'
    ],
    
    _handleEvent: function(evt) {
--- 5291,5297 ----
      'onfocusin', 'onfocusout', 'onactivate', 'onclick', 'onmousedown',
      'onmouseup', 'onmouseover', 'onmousemove', 'onmouseout', 'onload',
      'onunload', 'onabort', 'onerror', 'onresize', 'onscroll', 'onzoom',
!     'onbegin', 'onend', 'onrepeat'
    ],
    
    _handleEvent: function(evt) {
***************
*** 6329,6335 ****
  
  extend(_SVGWindow, {
    addEventListener: function(type, listener, capture) {
!     if (type == 'svgload' || type == 'SVGLoad') {
        this._onloadListeners.push(listener);
      }
    },
--- 6329,6335 ----
  
  extend(_SVGWindow, {
    addEventListener: function(type, listener, capture) {
!     if (type == 'load' || type == 'SVGLoad') {
        this._onloadListeners.push(listener);
      }
    },
