<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    
    <title>Arabidopsis nsSNP Help</title>
    <style type="text/css">
<!--
* {
    font-family: Helvetica, Arial, sans-serif;
}
.style1 {
	font-size: xx-large;
	font-weight: bold;
}
.style2 {
	font-size: x-large;
	font-weight: bold;
}
-->
    </style>
</head>

<body>
<p align="center" class="style1">Arabidopsis nsSNP Browser Help</p>
<p>&nbsp;</p>
<table width="900" border="0" align="center">
  <tr>
    <td width="80" rowspan="16" bgcolor="#8a6c89">&nbsp;</td>
    <td width="40" rowspan="16">&nbsp;</td>
    <td width="800">The Arabidopsis non-synonymous   single nucleotide polymorphism (<a href="http://snp.jbei.org/">At nsSNP</a>) browser provides a simple way to browse changes to  coding regions caused by SNPs in accessions or natural variants of Arabidopsis.</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td><div align="center" class="style2">Arabidosis SNP Data</div></td>
  </tr>
  <tr>
    <td>The data underlying At nsSNP is obtained  from the Arabidopsis <a href="http://1001genomes.org/">1001 Genomes Project</a> as processed SNP calls. We have created a pipeline to take these processed data and constructs pseudo chromosomes for each accession based on the Arabidopsis accession Col-0 genome sequence obtained  from <a href="http://www.arabidopsis.org/">TAIR</a> (currently TAIR9). From these pseudo chromosomes for each accession, we extract and compile proteins sets which are compared to the reference protein from TAIR.</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td><div align="center" class="style2">Using At nsSNP</div></td>
  </tr>
  <tr>
    <td><div align="center"><img src="AtnsSNP1.png" alt="AtnsSNP browser" width="743" height="421" /></div></td>
  </tr>
  <tr>
    <td><p>The data for a given AGI can be retrieved by  entering the code into the window on the top left and clicking the &quot;Retrieve&quot;  button. The current description for the protein is displayed (TAIR9) with the  protein represented as a bar scale with numbers indicating amino acid number.  The red/grey spheres indicate a nsSNP or amino acid substitution has occurred  at this point along the protein with the top track representing a consolidation  or summary of all nsSNPs in the protein. The letter in the red/grey sphere  corresponds to the substituted amino acid (1 letter code) as a result of the  nsSNP. The size of the red/grey sphere indicates the number of accessions or  natural variants that have contributed to a specific nsSNP or substitution. The  larger the red/grey sphere the more accessions have contributed to the change.</p></td>
  </tr>
  <tr>
    <td><div align="center"><img src="AtnsSNP2.png" width="757" height="527" /></div></td>
  </tr>
  <tr>
    <td>By employing (mouse click) the triangle in the dark grey &quot;Control Box&quot; a list of Arabidopsis accessions that contain nsSNPs to the particular protein are displayed.</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td><div align="center"><img src="AtnsSNP3.png" width="771" height="536" /></div></td>
  </tr>
  <tr>
    <td>The specific nsSNP from each Arabidopsis accession can be viewed by toggling the &quot;+&quot; symbol next to the accession name / code. In this instance Bur-0 has contributed one change and Bur-0 has contributed one change.</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td><div align="center"><img src="AtnsSNP4.png" width="794" height="619" /></div></td>
  </tr>
  <tr>
    <td><p>By panning the bar scale (protein) with the mouse  (grab and pan) and employing the mouse wheel to zoom, the specific substitutions  can be observed. In this example Bur-0 has a SNP that causes a substitution at  Met (106), resulting in an Ile; while Ped-0 contains a SNP that causes Thr at  position 141 to be substituted with a Ser.</p></td>
  </tr>
</table>
<p>&nbsp;</p>
</body>
</html>
